jetty9-base/pom.xml

```diff
@@ -112,7 +136,7 @@
               <goal>copy-dependencies</goal>
             </goals>
             <configuration>
-              <outputDirectory>${project.build.directory}/jetty-base/lib/gcp</outputDirectory>
+              <outputDirectory>${project.build.directory}/jetty-base/lib/gcloud</outputDirectory>
```

Comment:
I see above that you are still having the jetty-base jar copied into `${project.build.directory}/jetty-base/lib/gcp`. Is that intentional?